boards/shields/nrf700x_nrf54h20dk/nrf700x_nrf54h20dk.overlay

```diff
@@ -127,8 +28,67 @@
 		zephyr,wifi = &nordic_wlan0;
 		zephyr,bt-hci-ipc = "";
```

Comment:
Could you please remove it also? It removes my choice of bt-hci-ipc in the sample application, and I cannot overwrite it :)